docs: address uvicorn as a start option (#372)
kyma-project/kyma-companion: README.md

It is recommended to run Kyma Companion with Poetry because it activates and uses its virtual environment if not activated yet.

> As an alternative, you can run the application with the following command directly via Python, using [`uvicorn`](https://www.uvicorn.org/) instead of `FastAPI`:

Reviewer: ```suggestion
Alternatively, you can run the application directly using Python and [`uvicorn`](https://www.uvicorn.org/) instead of `FastAPI`. To do this, run the following command:
```

   ```bash
   python src/main.py
   ```

> If you need auto-reloading, you need to pass the `--reload` argument:

Reviewer: ```suggestion
To enable auto-reloading, pass the `--reload` argument:
```

> `python src/main.py --reload`

Reviewer: ```suggestion
    ```bash
    python src/main.pt --reload
    ```
```

> For IDEs (Pycharm, VS Code), you need to pass this argument in the run/debug configuration.

Reviewer: ```suggestion
For IDEs, such as Pycharm or VS Code, you must pass this argument in the run or debug configuration.
```

It is recommended to run Kyma Companion with Poetry because it activates and uses its virtual environment if not activated yet.

As an alternative, you can run the application with the following command directly via Python, using [`uvicorn`](https://www.uvicorn.org/) instead of `FastAPI`:

   ```bash
   python src/main.py
   ```

> If you need auto-reloading, you can add the corresponding key word argument in the `src/main.py`:
> 
>    ```python
>    if __name__ == "__main__":
>        uvicorn.run(app, host="0.0.0.0", port=8000, reload=True)
>    ```

Reviewer 2: ```suggestion
If you need auto-reloading, you need to pass the `--reload` argument:
`python src/main.py --reload`
For IDEs (Pycharm, VS Code), you need to pass this argument in the run/debug configuration.
```

Reviewer 2: I tried the `python src/main.py` which is not working when you curl the applcaiont. 
Shouldn't you use some like this `uvicorn main:app --reload`?
